tools/github_actions_install.sh

```diff
@@ -4,14 +4,14 @@
 # ($2) is an extra dependency (currently just nox on one run)
 set -eo pipefail
 export PYTHONUTF8=1
-if [[ "$SPHINX_VERSION" == "" ]]; then
-    SPHINX_INSTALL=""
+if [[ "$SPHINX_VERSION" == "rls" ]]; then
+    SPHINX_INSTALL="sphinx[test]"
 elif [[ "$SPHINX_VERSION" == "dev" ]]; then
     SPHINX_INSTALL="sphinx[test]@https://codeload.github.com/sphinx-doc/sphinx/zip/refs/heads/master"
 elif [[ "$SPHINX_VERSION" == "old" ]]; then
     SPHINX_INSTALL="sphinx[test]==6.1.0"
-else  # not used currently but easy enough
-    SPHINX_INSTALL="sphinx[test]==$SPHINX_VERSION"
+else  # for the "build site" jobs
+    SPHINX_INSTALL="sphinx<7.3"
```

Comment:
Since there is the new Ablog and Sphinx release seems we can remove the pin here too

```suggestion
    SPHINX_INSTALL="sphinx"
```